sed/loader/dask/loader.py

```diff
@@ -0,0 +1,104 @@
+"""
+module sed.loader.mpes, code for loading hdf5 files delayed into a dask dataframe.
+Mostly ported from https://github.com/mpes-kit/mpes.
+@author: L. Rettig
+"""
+import os
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Sequence
+from typing import Tuple
+
+import dask.dataframe as ddf
+
+from sed.loader.base.loader import BaseLoader
+from sed.loader.utils import gather_files
+
+
+class DaskLoader(BaseLoader):  # pylint: disable=too-few-public-methods
```

Comment:
I think renaming this module to something like GenericLoader makes more intuitive sense for the user for the features this loader supports.

Reply:
This can be done.